doc/whats-new.rst

```diff
@@ -14,11 +14,24 @@ What's New
 
     np.random.seed(123456)
 
-.. _whats-new.0.17.1:
+.. _whats-new.0.18.0:
 
-v0.17.1 (unreleased)
+v0.18.0 (6 May 2021)
 ---------------------
 
+This release brings a few important performance improvements, a wide range of
+usability upgrades, lots of bug fixes, and some new features. These include
+a new plugin API to add new engines, a new theme for the documentation, several
+new plot functions.
+
```

Comment:
```suggestion
a new plugin API to add new backend engines, a new theme for the documentation, 
curve fitting methods, and several new plotting functions.

```

Reply:
I need to merge it manually due to a conflict.